File: passbook-management-api/src/main/java/com/uit/passbook_management_api/entity/KhachHang.java
package com.uit.passbook_management_api.entity;

import jakarta.persistence.*;
import lombok.Data;

@Data
@Entity
@Table(name = "khach_hang")
public class KhachHang {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id; // SQL là BIGINT

    private String ten;

    @Column(unique = true)
    private String cmnd;

    @Column(name = "dia_chi")
    private String diaChi;

    private String sdt;

    // TODO: Thêm Getters / Setters
}

File: frontend-csharp/Models/KhachHangModel/KhachHang.cs
using System.Text.Json.Serialization;

namespace frontend_csharp.Models.KhachHangModel
{
    public class KhachHang
    {
        [JsonPropertyName("id")]
        public long Id { get; set; }

        [JsonPropertyName("ten")]
        public string Ten { get; set; }

        [JsonPropertyName("cmnd")]
        public string Cmnd { get; set; }

        [JsonPropertyName("diaChi")]
        public string DiaChi { get; set; }

        [JsonPropertyName("sdt")]
        public string Sdt { get; set; }
    }
}